addons/hr/models/mail_alias.py

```diff
@@ -13,7 +15,9 @@ class Alias(models.Model):
 
     def _get_alias_bounced_body_fallback(self, message_dict):
         if self.alias_contact == 'employees':
-            return _("""Hi,<br/>
+            return Markup(
+                _("""<p>Hi,<br/>
```

Comment:
Why add the surrounding paragraphs?

Reply:
Old habit to always return something in an html node. Jinja changed my life.

Reply:
:cry:

Reply:
As it is embedded "as it" inside a div, I think it is better to have a p around. And because.